### DISCUSSION TITLE
Advanced marker support

### DISCUSSION THREAD

**Discussion by Willham12** [react-native-maps/react-native-maps#5257] [Feature requests]
Anyone here is working on this?
https://developers.google.com/maps/documentation/ios-sdk/advanced-markers/add-marker
https://developers.google.com/maps/documentation/android-sdk/advanced-markers/add-marker

I have a app with custom markers and running in poor performance working with trackViewChanges and best practises to reduce re-rendering. Maybe this can be the solution. Otherwise i have to create my own maps package with native views that matches my need.

**Comment by mr-tm** [comment]
Yes. There was a pull request: https://github.com/react-native-maps/react-native-maps/pull/5039
I pinged maintainers, but not sure if they will notice.

**Comment by salah-ghanim** [comment]
@mr-tm it has to wait unfortunately until Fabric is supported, given that it's blocking so many people from updating to rn 0.76+

but thanks for the contribution :)
